README_NEW.md

```diff
@@ -0,0 +1,349 @@
+# Go backend for MLflow
+
+In order to increase the performance of the tracking server and the various stores, we propose to rewrite the server and store implementation in Go.
+
+
+# Topics
+* [Installation](#installation)
+* [Mage](#mage)
+* [Development Setup](#development-setup)
+  * [Installation](#installation)
+  * [Run tests](#run-tests)
+  * [Tests state](#tests-state)
+* [Common information](#common-information)
+  * [MLflow source code](#mlflow-source-code)
+  * [Protos](#protos)
+  * [Request validation](#request-validation)
+  * [Data access](#data-access)
+  * [Linting](#linting)
+* [Usage](#usage)
+  * [Go Server](#go-server)
+  * [Building the Go binary](#building-the-go-binary)
+  * [Client-side Go implementation](#client-side-go-implementation)
+  * [Go store in Python](#go-store-in-python)
+* [Misc](#misc)
+  * [Debug Failing Tests](#debug-failing-tests)
+  * [Supported endpoints](#supported-endpoints)
+  * [Targeting Local Postgres in Python Tests](#targeting-local-postgres-in-python-tests)
+
+## Installation
+
+This package is not yet available on PyPI and currently requires the [Go SDK](https://go.dev) to be installed.
```

Comment:
Why don't we push builds to PyPI so this is simpler?

Reply:
We first need to move to https://github.com/mlflow/mlflow-go and set it up over there.
Might require helps from Databricks.